### PR TITLE
Fix for NGRAPH_DISTRIBUTED

### PR DESCRIPTION
**Pull request by vishakha-nervana**

### CODE REVIEW

File: ngraph_bridge/ngraph_executor.cc [hunk 1]
```diff
@@ -295,12 +295,7 @@ NGraphExecutor::CreateCallback(const std::string signature,
   // Serialize to nGraph if needed
   if (std::getenv("NGRAPH_ENABLE_SERIALIZE") != nullptr) {
     std::string file_name = "tf_function_" + m_node_name + ".json";
```
Comment (shresthamalik) [inline ngraph_bridge/ngraph_executor.cc]: is this file_name being used anywhere?

Comment (vishakha-nervana) [reply ngraph_bridge/ngraph_executor.cc]: No, it is not.

Comment (shresthamalik) [reply ngraph_bridge/ngraph_executor.cc]: Please remove it then.

Comment (vishakha-nervana) [reply ngraph_bridge/ngraph_executor.cc]: Done

**Pull request merged**